### PR TITLE
Fix CI failures

### LINKED ISSUE #12840: CI failures
## Description

- [ ] Example browser-require test is broken: https://github.com/jupyterlab/jupyterlab/runs/7465263087?check_suite_focus=true
- [ ] UI tests for extension manager is broken (due to yet another change in the package ordering - we should try the new Playwright feature [recording the network](https://playwright.dev/docs/release-notes#network-replay))

### PR DESCRIPTION
**Pull request by fcollonval**
## References

Fixes #12840

## Code changes

Upgrade @jupyterlab/services downloaded by requireJS
Update the example to work with the latest version
Fake more npmjs.org request for UI tests

## User-facing changes

N/A

## Backwards-incompatible changes

N/A

### CODE REVIEW

File: galata/test/documentation/data/extensions-search-all.json [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+          "repository": "https://github.com/jupyterlab/jupyterlab",
+          "bugs": "https://github.com/jupyterlab/jupyterlab/issues"
+        },
+        "author": { "name": "Project Jupyter" },
+        "publisher": {
+          "username": "fcollonval",
+          "email": "fcollonval@gmail.com"
+        },
+        "maintainers": [
+          { "username": "3coins", "email": "pyjain@gmail.com" },
```
Comment (jtpio) [inline galata/test/documentation/data/extensions-search-all.json:86]: Maybe we could use placeholder emails here? (even though they are publicly available from npm)

**Review by jtpio — APPROVED** [review]
Thanks!

Looks like all CI checks are now passing :tada: 

Just left a comment about using placeholder email addresses instead of the real ones.

File: galata/test/documentation/data/extensions-search-all.json (hunk 1 above)
Comment (fcollonval) [reply galata/test/documentation/data/extensions-search-all.json:86]: Oh yes, I copied the dump without a second thought. Will move to dummy users.

Comment (jtpio) [reply galata/test/documentation/data/extensions-search-all.json]: :+1:

**Comment by jtpio** [comment]
Thanks!

**Pull request merged**